docs/UserGuide.md

```diff
@@ -237,20 +245,32 @@ Format: `sort KEYWORD`
 
 * Sorts the list of clients based on specified keyword
 * Keyword must be from client details
-* (v1.2) current supported sort metrics include `name, phone number, address`
+
+
+
+Sorting **KEYWORDS** are shown in the table below: 
+
+| KEYWORD          | Function                                                                                                             |
+|:-----------------|:---------------------------------------------------------------------------------------------------------------------|
+| `default`        | Based on when a client was added, from oldest to newest                                                              |
+| `name`           | By alphabetical order ie. "a" to "z"                                                                                 |
+| `email`          | Clients using the same email platform are grouped together, thereafter, alphabetical order is used within each group |
+| `phone`          | By numerical order of the phone number                                                                               |
+| `birthday`       | From oldest to youngest   |                                                                                            
+| `income`         | From highest to lowest income |
+| `risk appetite`  | From highest to lowest ie. "H" to "M" to "L" |
+
 
 Examples:
 * `sort name` will display your address book in alphabetical order based on their name
 * `sort phone` will display your addresss book in numerical order based on their phone numbers
 
-Sorting **KEYWORDS** include: 
-* `default` (the default order is based on when a client is added, from oldest to newest)
-* `name` (by alphabetical order)
-* `email` (clients using the same email platforms are grouped together, thereafter, alphabetical order is used within each group)
-* `phone` (by numerical order of phone number)
-* `birthday` (from oldest to youngest)
-* `income` (from highest to lowest income)
-* `risk appetite` (from highest to lowest)
+
+Below is an example of what you will expect to see when you call `sort email`:
+
+:exclamation: Add picture here
```

Comment:
Just to check, is this placeholder intended?

Reply:
yes sir i haven't screenshot yet